packages/lib-user/src/components/shared/AllProjects/AllProjects.js

```diff
@@ -0,0 +1,73 @@
+import { Box, Text } from 'grommet'
+import styled from 'styled-components'
+import { arrayOf, bool, number, shape, string } from 'prop-types'
+import ProjectCard from '@zooniverse/react-components/ProjectCard'
+import Loader from '@zooniverse/react-components/Loader'
+import { useTranslation } from '../../../translations/i18n.js'
+
+const StyledBox = styled(Box)`
+  list-style: none;
+  margin-block-start: 0;
+  padding-inline-start: 0;
+  column-gap: 20px;
+  row-gap: 20px;
+`
+
+function AllProjects({ error, loading, projects }) {
+  const { t } = useTranslation()
+
+  return (
+    <>
+      {loading ? (
+        <Box fill align='center' justify='center'>
+          <Loader />
+        </Box>
+      ) : projects?.length === 0 ? (
+        <Box fill align='center' pad='medium'>
+          <Text>{t('AllProjects.noProjects')}</Text>
+        </Box>
+      ) : error ? (
+        <Box fill align='center' pad='medium'>
+          <Text>{t('AllProjects.error')}</Text>
+        </Box>
+      ) : (
+        <StyledBox forwardedAs='ul' direction='row' wrap>
+          {projects.map(project => {
+            return (
+              <li key={project?.id}>
+                <ProjectCard
+                  badge={project?.count}
+                  description={project?.description}
+                  displayName={project?.display_name}
+                  href={`https://www.zooniverse.org/projects/${project?.slug}`}
+                  imageSrc={project?.avatar_src}
+                  size='large'
```

Comment:
I'll chat with Sean about how to handle these project cards on smaller screens like a phone. For now, they're all large to allow for 5 cards per row at the largest layout.

Reply:
When I view this page in the same manner as the screenshot on a large monitor, I see five cards across. There must be a 1-pixel difference between the way our browsers render borders or padding that causes the fifth to bump to the next row on yours. Referring to Figma, I tried to create 20px gap between project cards with 50px ContentBox padding, but that doesn't work easily in an actual browser so I'll reduce the ContextBox padding so everyone sees 5-across on large monitors 👍

Reply:
Interesting! Solution sounds good.